data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -84,6 +84,16 @@ impl TraceExporterOutputFormat {
             },
         )
     }
+
+    #[cfg(feature = "test-utils")]
+    // This function is only intended for testing purposes so we don't need to go to all the trouble
+    // of breaking the uri down and parsing it as rigorously as we would if we were using it in
+    // production code.
+    fn add_query(&self, url: &Uri, query: &str) -> Uri {
```

Comment:
/suggestion

as you mentioned that this is for testing only, can this be moved out? 

so we don't end up using in prod code accidentally?

Reply:
It's behind the `test-utils` feature so it won't be included in production code.